NitroxServer/Serialization/World/WorldPersistence.cs

```diff
@@ -214,5 +213,49 @@ public World CreateWorld(PersistedWorldData pWorldData, ServerGameMode gameMode)
 
             return world;
         }
+
+        private void UpgradeSave()
+        {
+            string saveDir = config.SaveName;
+
+            SaveFileVersion saveFileVersion = Serializer.Deserialize<SaveFileVersion>(Path.Combine(saveDir, $"Version{FileEnding}"));
+
+            // SaveFileVersion structure was updated in V1.5.0.0
+            // This can be removed with V1.6.0.0 or later
+            if (File.ReadAllText(Path.Combine(saveDir, $"Version{FileEnding}")).Contains("BaseDataVersion"))
+            {
+                saveFileVersion = new SaveFileVersion(new Version(1, 4, 0, 0));
+            }
+
+            if (saveFileVersion.Version == NitroxEnvironment.Version)
+            {
+                Log.Info("Save files are already at the newest version");
```

Comment:
Don't know if we should keep this Log. Would be executed on every server start...

Reply:
Yes I agree, it was something I was thinking about removing as you are right does show each startup.